code/API_definitions/application-profiles.yaml

```
@@ -166,7 +188,7 @@ paths:
     delete:
       security:
         - openId:
-            - connectivity_insights:application_profile:write
+            - application-profile:delete
```

As `api-name` is application-profiles
```suggestion
            - application-profiles:delete
```